synapse/storage/databases/main/event_federation.py

```diff
@@ -1123,10 +1147,16 @@ async def get_backfill_events(
         )
         events = await self.get_events_as_list(event_ids)
         return sorted(
-            events, key=lambda e: (-e.depth, -e.internal_metadata.stream_ordering)
+            events, key=lambda e: (-e.depth, -e.internal_metadata.stream_ordering)  # type: ignore[operator]
```

Comment:
Reason: `synapse/storage/databases/main/event_federation.py:1150: error: Unsupported operand type for unary - ("Optional[int]")  [operator]`